lib/pages/AthletePage.dart

```diff
@@ -1264,15 +1691,15 @@ class _AthletePageState extends State<AthletePage> {
                             alignment: Alignment.centerLeft,
                             child: Text("Details",
                                 style:
-                                    textStyle(Colors.white, 24, false, false))),
+                                textStyle(Colors.white, 24, false, false))),
                         Divider(thickness: 1, color: greyTextColor),
                         Row(
                             mainAxisAlignment: MainAxisAlignment.spaceBetween,
                             children: <Widget>[
                               Text("Sport / League",
                                   style: textStyle(
                                       greyTextColor, 20, false, false)),
-                              Text("American Football / NFL",
+                              Text("BaseBall / MLB",
```

Comment:
Use the sport property in the athlete model

Reply:
done